client/src/pages/private/participant-details/modals/AssignCohortDialog.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Box, Grid, Typography } from '@material-ui/core';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+
+import { getCohortName } from '../constants';
+import { Dialog, Button } from '../../../../components/generic';
+
+export const AssignCohortDialog = ({ isOpen, onClose, onSubmit, participant, selectedCohort }) => {
```

Comment:
```suggestion
export const AssignCohortSiteDialog = ({ isOpen, onClose, onSubmit, participant, selectedCohort }) => {
```
Does this make more sense?